### DISCUSSION TITLE
Welcome to Annotations discussions!

### DISCUSSION THREAD

**Discussion by danhendersonede** [cvs-health/annotations#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by gsilver** [comment]
Hi, on the Introducing the Jan 23, 2024 [Accessibility Annotations Kit for iOS from CVS Health® Inclusive Design](https://medium.com/cvs-health-tech-blog/introducing-the-accessibility-annotations-kit-for-ios-from-cvs-health-inclusive-design-19be1bf2fcc5) post you mention that "We’ll be publishing a follow up article in a few weeks where we dive into a few examples of how we use it." - is there any news on this? May have missed the following article. Thanks for the good work

**Reply by danhendersonede** [reply]
Hey there, @gsilver! Unfortunately that never got published, however if you'd like to chat more about it I'm happy to [connect on LinkedIn](https://www.linkedin.com/in/danhendersonede/).

Thanks,
Daniel
